### DISCUSSION TITLE
Discussion for the biweekly meeting (2023/04/03)

### DISCUSSION THREAD

**Discussion by johnlanni** [alibaba/higress#270] [General]
## 周会纪要

1. Higress 用户文档结构讨论，前期主要是内容完善：
    a. 参考手册:  Ingress注解&CRD配置的完整说明
    b. 插件使用：所有插件的配置说明文档，自定义插件开发配置说明文档
    c. 最佳实践：给出具体应用场景的例子，例如 Higress 对接 Spring Cloud/Dubbo 服务
2. Higress 文档在内容完善后，讨论到进一步需要做的：
    a. 国际化：完成全部文档内容的翻译
    b. Demo完善：提供类似 istio bookinfo 的 demo 代码，可以运行测试所有功能
3. 介绍 DeGraphql wasm 插件的实现方式以及可以用于e2e测试的配置说明，已经被社区同学认领;  GraphQL 对比 REST 有很多优势，Higress 后续会探索基于 GraphQL 对接多种数据源(微服务/DB/FC等)的能力，欢迎有兴趣的同学沟通讨论
4. wasm 插件生态是 Higress 会重点推进，目前基于 go 开发是最简单的，也支持 rust 等语言开发，对插件开发感兴趣的同学也欢迎沟通，插件开发过程，我们也会提供很多帮助

## 社区进展同步

1. 支持了基于静态IP或者DNS发现的服务发现机制，Higress 控制台交互也已经发布
2. 支持识别 K8s 版本选择对应的控制面认证方式(JWT policy)
3. 修复 higress.io/destination 注解对于 Dubbo 服务端口的解析错误

**Comment by tomkerkhove** [comment]
Thanks for the notes! I'm definately interested in the WASM plug-ins.
I'm not familiar with them yet, but curious to see how I can use .NET to write those plug-ins in the future.

**Reply by johnlanni** [reply]
> Thanks for the notes! I'm definately interested in the WASM plug-ins. I'm not familiar with them yet, but curious to see how I can use .NET to write those plug-ins in the future.

If the plug-in developed by .Net follows this [ABI](https://github.com/proxy-wasm/spec), it can be used for envoy execution
